assets/src/sass/objects/_header.scss

```diff
@@ -211,6 +211,10 @@ $header-bp-mw: 1049px;
 		z-index: 2;
 		margin-top: rem(30);
 		width: 100%;
+
+		body.hide-page-title & {
```

Comment:
Hiding the `h1` inside `header-content` containing the page title only may be a better option, as it's currently hiding also an `<h2 class="h4 eyebrow">` which has a back link on some pages.

```suggestion
		body.hide-page-title & h1 {
```

I tested the suggested solution here and it's looking good.

![image](https://user-images.githubusercontent.com/90911997/223590898-73f236d8-9c69-4189-a580-a56a6f0c4c41.png)